### PR TITLE
Implement the GET /rules endpoint

### PR DESCRIPTION
**Pull request by Matthew-Collins-Defra**
https://eaflood.atlassian.net/browse/FCEP-46

### CODE REVIEW

File: test/index.test.js [hunk 1]
```diff
@@ -46,6 +46,16 @@ describe('Web test', () => {
     expect(response.statusCode).toBe(200)
   })
 
+  test('GET /rules route works', async () => {
```
Comment (djbingham) [inline test/index.test.js]: Should we put tests for each route in a dedicated file under `test/routes/`? The healthcheck tests are already there.

File: test/services/rulesService.test.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+
+const rulesService = require('../../server/services/rulesService')
+describe('rulesService', () => {
+  test('rules service returns a rules object containing an array', async () => {
+    expect.assertions(1)
+    const rules = await rulesService.get()
+    expect(Array.isArray(rules.rules)).toEqual(true)
+  })
+
+  test('rules service object contains an array there the first element has an id of 1', async () => {
```
Comment (paulsimonandrews) [inline test/services/rulesService.test.js]: Typo: "there" -> "where"

File: test/index.test.js (hunk 1 above)
Comment (Matthew-Collins-Defra) [reply test/index.test.js]: Yes, good idea.

Comment (paulsimonandrews) [reply test/index.test.js]: Agreed

**Review by paulsimonandrews — APPROVED** [review]
LGTM

**Pull request merged**